File: Java/SivitasAkademik.java
public class SivitasAkademik extends Human{
    
    private String asal_univ;
    private String email;

    public SivitasAkademik(){
		this.asal_univ = "";
        this.email = "";
	}

    public SivitasAkademik(String NIK, String name, String gender, String asal_univ, String email){
		super(NIK, name, gender);
        this.asal_univ = asal_univ;
        this.email = email;
	}

    // Untuk asal_univ
	public String getAsal_univ(){
		return this.asal_univ;
	}

	public void setAsal_univ (String asal_univ){
		this.asal_univ = asal_univ;
	}

    // Untuk Nama
	public String getEmail(){
		return this.email;
	}

	public void setEmail (String email){
		this.email = email;
	}
}

File: Python/SivitasAkademik.py
from Human import Human

class SivitasAkademik(Human):
    
    # Inisialisasi variabel untuk setiap objek dari kelas
    asal_univ = ""
    email = ""
    
    def __init__(self, asal_univ = "", email = "", NIK = "", name = "", gender = ""):
        super().__init__(NIK, name, gender)
        self.asal_univ = asal_univ
        self.email = email
        
    # Getter dan Setter untuk Asal Universitas
    def get_Asal_univ(self):
        return self.asal_univ
    
    def set_Asal_univ(self, asal_univ):
        self.asal_univ = asal_univ
        
    # Getter dan Setter untuk Email
    def get_Email(self):
        return self.email
    
    def set_Email(self, email):
        self.email = email